content/id/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Memahami Konsep Objek-Objek yang Ada pada Kubernetes
```

Comment:
Capital typo in `Ada`.

Reply:
Hmm I was wondering if the only things that left non capitalized is conjunctions cc @girikuncoro

Reply:
I think we should go with standard Indonesian language for title, which in this case lowercase `ada`